### DISCUSSION TITLE
Zoom

### DISCUSSION THREAD

**Discussion by revidovich** [Leaflet/Leaflet#8100] [Q&A] [answered]
zoom gives just numbers from 0 to 18. tell me please what scale is it?

**Comment by IvanSanchez** [comment, marked as answer]
Zoom zero means that 256 pixels cover the span of the equator on a EPSG:3857 map projection. That means 40075016,68 EPSG:3857 units per 256 pixels, or ~156543,034 units per pixel - that's both horizontally and vertically.

Each zoom level is a factor of two of the previous one. Therefore, the scale for any given zoom level is (`156543,034 * 2^z`/px), always expressed in EPSG:3857 units.

**Reply by revidovich** [reply]
thanks for the detailed answer!
